BibBuddy/app/src/main/java/de/bibbuddy/BookNotesView.java

```diff
@@ -320,6 +325,21 @@ private Bundle createNoteBundle(NoteItem item) {
     return bundle;
   }
 
+  private void fillBookData() {
+    Bundle bundle = this.getArguments();
+    Book book = bookDao.findById(bookId);
```

Comment:
```suggestion
    Book book = bookModel.getBookById(bookId);
```
The fragment needs to handle the data through a bookModel and should note use the DataObjects by itself. Remove all similar dataObjects-usages.

Reply:
I was actually looking for that method and couldn't find it in bookModel, sry! Thanks for the suggestion, done :)